README.Rmd

```diff
@@ -195,10 +195,10 @@ Rust toolchain
  
 #### Development workflow
 
-Assuming the system dependencies have been met (above), the typical **rpolars**
+Assuming the system dependencies have been met (above), the typical **polars**
 development workflow is as follows: 
 
-**Step 1:** Fork the **rpolars** repo on GitHub and then clone it locally.
+**Step 1:** Fork the **polars** repo on GitHub and then clone it locally.
```

Comment:
Again, to avoid ambiguity, I would say: "..Fork the R **polars repo..."

Also: This is more to do with Step 2+ (not shown here) but is this workflow still the same even if the repo structure has changed? Aren't we using the `pola-rs/rpolars/package` suggested by @eitsupi? (Or did I miss something?) If so, wouldn't they at least need to cd one directory deeper, i.e. `cd r-polars/package`...